### ISSUE TITLE
How to generate open api file using the cli tool

### ISSUE THREAD

**Issue by ShahOdin** [disneystreaming/smithy4s#571]
I'm looking at: https://disneystreaming.github.io/smithy4s/docs/overview/cli and trying to generate the open api from the smithy files. in a custom directory taking the structure introduced in the quick start example as the starting point. 

For a service `PlaygroundService` with multiple operations `Choice,CreateCustomer,Hello`:

```bash
playground % tree -I target

.
└── src
    └── main
        └── smithy
            ├── Choice.smithy
            ├── CreateCustomer.smithy
            ├── Hello.smithy
            └── PlaygroundService.smithy
└── temp
```

- I tried the following from the top directory ^:

  ```bash
  smithy4s generate \
  --output ./temp/ \
  --resource-output ./temp \
  --discover-models
  ```

  I simply get blank files generated under `temp/META-INF`.

- I tried `cd` ing to `./src/main/smithy` and re-ran the above. again. nothing.
- I played around with the `-- repositories` and `-- dependencies` options with no luck. 

What am I missing? Would be great if you could add a minimal example in the docs as well.

**Comment by kubukoz** [comment]
I think you need to pass `./src` or, to be more precise, `./src/main/smithy`, as the positional argument to `generate`. You also probably don't need `--discover-models`.

**Comment by ShahOdin** [comment]
D'oh! that does it @kubukoz thank you. :) shall I still keep the issue open for the docs to make this clearer?

**Comment by kubukoz** [comment]
yeah, I think that's a good idea. Would you like to make that change in the docs?

**Comment by Baccata** [comment]
probably yes

**Comment by kubukoz** [comment]
I think we should at least log something. An error is fine as long as it's only for CLI usage, not programmatic codegen usage

**Comment by daddykotex** [comment]
I closed my PR because in the `series/0.17`, we have a default dependencies, so my check was always false :/ Maybe a warning we have the model in hand and we realize that it contains no shapes is enough

**Comment by kubukoz** [comment]
Is there anything else we want to do here or can the ticket be closed?
